static/js/components/ExpandedPostDisplay.js

```diff
@@ -153,17 +194,19 @@ export default class ExpandedPostDisplay extends React.Component<*, void> {
     return (
       <div className="post-summary expanded">
         <div className="summary">
-          <ProfileImage
-            profile={{
-              name:        post.author_name,
-              image_small: post.profile_image
-            }}
-            useSmall={true}
-          />
           <div className="post-title">{formatPostTitle(post)}</div>
```

Comment:
I think the font size of this should increase to match the mockup. I'm uncertain as to what exact size it should be.

Reply:
yeah, it's 28px on the Zeplin thing so I'll change it to that